build-utils/rebuild_inbound_admin_proxy_image.sh

```diff
@@ -2,4 +2,4 @@
 set -euxo pipefail
 
 ./build-utils/base_rebuild_single_docker_image.sh dataland_inbound_admin_proxy ./dataland-inbound-admin-proxy/Dockerfile \
-        ./dataland-pgadmin/
+        ./dataland-inbound-admin-proxy/
```

Comment:
Die Parameterliste wird doch nur für die sha Berechnung gebraucht und bei dieser Berechnung werden doch eh alle Dateien in einem Ordner genommen sprich wenn man in den Parametern Dateien oder Unterordner angibt von denen das übergeordnete Verzeichnis auch in der Liste ist kann man sich das eigentlich sparen. Das sollte ja nur zu duplizierten Einträgen in der sha Berechung führen. Diese Duplikate sollte man vermutlich eh mit "sort -u" rausfiltern

Reply:
haben wir besprochen: Das dockerfile muss explizit angegeben werden, damit docker build weiß, wo das dockerfile liegt